testsuite/domain/src/test/java/org/jboss/as/test/integration/domain/HostExcludesTestCase.java

```diff
@@ -208,7 +210,11 @@ private enum ExtensionConf {
         // to the internal mpExtensions Set defined on this class.
         // Don't add here extensions supplied only by the wildfly-preview-feature-pack because we are not tracking different releases
         // of wildfly preview. In such a case, add them to previewExtensions set defined below.
-        CURRENT(MAJOR, WILDFLY_27_0, null, getCurrentRemovedExtensions());
+        CURRENT(MAJOR, WILDFLY_27_0, Arrays.asList(
+                "org.wildfly.extension.micrometer"
+        ),
+                getCurrentRemovedExtensions()
+        );
```

Comment:
Notice the comment about this enum:

```
If an extension is added to this enum, also check if it is supplied only by wildfly-galleon-pack. If so, add it also
to the internal mpExtensions Set defined on this class.
```

You also add the "org.wildfly.extension.micrometer" to the `mpExtensions` HashSet.